tests/installation/await_install.pm

```diff
@@ -125,7 +125,7 @@ sub run {
             send_key 'alt-o';    # ok
             next;
         }
-        if (get_var('LIVECD') and match_has_tag('screenlock')) {
+        if (get_var('LIVECD') and match_has_tag('screenlock') and !check_screen('blackscreen', 0)) {
```

Comment:
Use `match_has_tag` instead of `check_screen` to not run the needle detection again.

Reply:
Done.